src/theme.ts

```diff
@@ -8,5 +8,46 @@ export const theme: DefaultTheme = {
   colors: {
     primary: '#007896',
     secondary: '#163150'
+  },
+  fontSizes: ['10px', '12px', '16px', '18px', '20px', '40px'],
+  lineHeights: ['16px', '24px', '48px'],
+  text: {
+    heading: {
+      fontFamily: 'body',
+      fontSize: 5,
+      lineHeight: 3,
+      fontWeight: 700,
+      color: 'DARK_BLUE'
+    },
+    subHeading: {
+      fontFamily: 'body',
+      fontSize: 4,
+      lineHeight: 2,
+      color: 'DARK_BLUE',
+      fontWeight: 700
+    },
+    body: {
+      fontFamily: 'body',
+      fontSize: 2,
+      lineHeight: 1,
+      color: 'BODY'
+    },
+    muted: {
+      fontFamily: 'body',
+      fontSize: 2,
+      lineHeight: 1,
+      color: 'BLUE_GREY'
+    },
+    error: {
+      fontFamily: 'body',
+      fontSize: 2,
+      lineHeight: 1,
+      color: 'RED'
```

Comment:
The colours here aren't specified in the theme. We should also use `primary`, `secondary`, ... instead of colour names I think, but I'm not exactly sure what that would look like.

Reply:
I would like to rename the colors, but I don't know how we would convert the current colors to a theme color scheme. We might need design to come up with this?

Reply:
```
COLORS: {
  TEXT: {
    primary: BODY,
    secondary: DARK_BLUE,
    discrete: BLUE_GREY,
    warning: RED
  }
}

```